clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -407,7 +407,7 @@
                                         CommonClientConfigs.CONNECTIONS_MAX_IDLE_MS_DOC)
                                 .define(INTERCEPTOR_CLASSES_CONFIG,
                                         Type.LIST,
-                                        null,
+                                        "",
```

Comment:
Isn't this weird? Seems like the default should be an empty list and not an empty string?

Reply:
Yes indeed it would make sense.

Reply:
Agreed it is odd. I suggested it because we did it elsewhere, but it looks like the code would actually support using an empty list directly here.

Reply:
Was there any reason not to change the default to be `emptyList()`?